front/src/modules/ui/modal/components/Modal.tsx

```diff
@@ -2,6 +2,14 @@ import React from 'react';
 import styled from '@emotion/styled';
 import { motion } from 'framer-motion';
 
+const StyledContainer = styled.div`
+  align-items: center;
+  color: ${({ theme }) => theme.font.color.primary};
+  display: flex;
+  flex-direction: column;
+  padding: ${({ theme }) => theme.spacing(10)};
```

Comment:
We should not enforce padding on here: It's breaking other Modals. This shoud only be in the DeleteWorkspace modal